.gitignore

```diff
@@ -23,4 +23,5 @@ yarn-error.log*
 
 /.idea
 .env
-.next
\ No newline at end of file
+.next
+package-lock.json
```

Comment:
Can you add a newline here to get rid of the (-) error?

Reply:
We should add `eol-last: ["error", "always"]` to eslint as well then to avoid this in the future @topanb11

Reply:
Added it to the eslint file